### PR TITLE
Create FOXE-FOXEERF745V3.config

### PR DESCRIPTION
**Pull request by Johnsnow-foxeer**
Would like to add Foxeer F745 AIO V3 target configuration based on  ICM-42688-P gyro, thank you

### CODE REVIEW

File: configs/default/FOXE-FOXEERF745V3.config [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Betaflight / STM32F745 (S745) 4.1.0 Sep 25 2020 / 01:23:16 (064ca9b75) MSP API: 1.42
+
+board_name FOXEERF745V3_AIO
```
Comment (haslinghuis) [inline configs/default/FOXE-FOXEERF745V3.config:3]: The name of the file has to reflect the MFR - board_name

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF745V3.config (hunk 1 above)
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF745V3.config]: corrected it, thank you

**Review by sugaarK — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Review by Asizon — APPROVED** [review]
Aproved supposing that it is an aio board with integrated ESC.

**Comment by sugaarK** [comment]
> Aproved supposing that it is an aio board with integrated ESC.

Sure is

**Pull request merged**